standalone/standalone.py

```diff
@@ -655,6 +658,14 @@ def show(
     ),
     envvar="JUDGE_SERVING_MODEL_API_KEY",
 )
+@click.option(
+    "--judge-serving-model-ca-cert",
+    type=str,
+    help=(
+        "Name of the Kubernetes ConfigMap containing the serving model CA cert."
+        "The expected key name is 'ca-bundle.crt'."
```

Comment:
this should be parameterized, note that in odh/rhoai the user will need to put `odh-ca-bundle.crt`, not something we want to hardcode here

Reply:
*Sigh.* I was hoping to dodge more flags :)